receiver/azuremonitorreceiver/config.go

```diff
@@ -16,6 +16,7 @@ import (
 const (
 	azureCloud           = "AzureCloud"
 	azureGovernmentCloud = "AzureUSGovernment"
+	azureChinaCloud      = "AzureChinaCloud"
```

Comment:
Would be good to add support for "AzureGermanCloud" as well, i'll open a follow up issue